applications/GeoMechanicsApplication/custom_workflows/time_loop_executor.hpp

```diff
@@ -72,6 +72,7 @@ class TimeLoopExecutor : public TimeLoopExecutorInterface
             NewEndState = RunCycleLoop(NewEndState);
             mStrategyWrapper->AccumulateTotalDisplacementField();
             mStrategyWrapper->FinalizeSolutionStep();
+            mStrategyWrapper->ComputeIncrementalDisplacementField();
```

Comment:
Very nice you thought about the C++ route. Would it be an idea to also add it to the settlement workflow test (I know it's not running in the pipeline, but we can still use it locally to test)

Reply:
Thank you for the catch, should be before the FinalizeSolutionStep as in python.